dashboard/app/models/lesson.rb

```diff
@@ -663,6 +663,8 @@ def next_level_for_lesson_extras(user)
   end
 
   def next_level_path_for_lesson_extras(user)
+    return script_path(script) if
+    script.show_unit_overview_between_lessons?(user)
```

Comment:
I put the logic here because this is the method that returns the path. The new case I added is also returning a path, the script path for the script, so it felt conceptually similar to me. 

Good catch - fixed the formatting thanks! The inline if was a couple of characters over our line wrap limit 😄